### PR TITLE
Fairly Sizable Fed Lore Drop

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Combs through basically everything related to the Trans-Solar Federation to make sure it's lore compliant and flows well.

- Many pieces of Fed gear have been given updated descriptions and names.

- The vaguely named "specops" marines have been renamed to MARSOC.

- Renames ARG magazines to M26 mags because i forgor.

- probably more stuff i forgot

## Why It's Good For The Game
Lore being consistent with game is good. Descriptions are good. Cool names are good.

## Testing

- Entered game
- Became several Federation mobs
- Spawned many Federation items
- All good here boss
<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
 
  <hr>

## Changelog

:cl:
tweak: Updated a whole lotta fed lore
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
> * All uses of "solar federation" in naming have been shortened to just "Federation", as "solar federation" is an incorrect name and the full "Trans-Solar Federation" is a bit of a mouthful.

my main worry with this is, federation of what? the word "federation" doesn't convey any information.
i'd personally suggest using the `T.S.F` abbreviation, as it conveys what they are, while being short. And is inline with irl similarish groups, like the C.I.A, and F.B.I

**Comment by PopeDaveThe3th** [comment]
> > * All uses of "solar federation" in naming have been shortened to just "Federation", as "solar federation" is an incorrect name and the full "Trans-Solar Federation" is a bit of a mouthful.
> 
> 
> 
> my main worry with this is, federation of what? the word "federation" doesn't convey any information. especially since we have atleast two "federations" within lore(wizard federation)
> 
> i'd personally suggest using the `T.S.F` abbreviation, as it conveys what they are, while being short. And is inline with irl similarish groups, like the C.I.A, and F.B.I 

Hrm. I see where's you're coming from. I'll look into that later today to see if it sounds right. I will say that I dislike the Wizard Federation as a name since it's boring and uninspired but that's for another time.

**Comment by Fordoxia** [comment]
Honestly there's several roles where fully spelling out Trans-Solar Federation makes sense (representative, traders...), and the rest should be using contractions at the very least to support easy distinguishing of the TSF assets from any other federations current or yet to exist.

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -437,7 +437,8 @@
 	multi_sprite_step = AMMO_BOX_MULTI_SPRITE_STEP_ON_OFF
 
 /obj/item/ammo_box/magazine/m556/arg
-	name = "\improper ARG magazine (5.56mm)"
+	name = "\improper M26 STNEQ magazine (5.56mm)"
```
Comment (Fordoxia) [inline code/modules/projectiles/ammunition/magazines.dm:440]: "STNEQ" is an orphan acronym that is not explained anywhere. Dropping it directly into Google fails to demystify it.

The actual rifle that it goes into is simply called the M26A2 assault rifle, so this also does not help identify the meaning.

**Review by Fordoxia — APPROVED** [review]
Looks federal to me.

**Review by lewcc — APPROVED** [review]
Changes look good to me.

**Review by S34NW — APPROVED** [review]

**Pull request merged**